server/test/unit/plugins/util/test_verification.py

```diff
@@ -49,6 +36,10 @@ def test_SHA256_to_sha256(self):
 
         self.assertEqual(checksum_type, 'sha256')
 
+    def test_invalid_type_raises_coded_exception(self):
+        self.assertRaises(PulpCodedException,
+                          verification.sanitize_checksum_type, 'not_a_real_checksum')
```

Comment:
That competes with my new checksum library `didnt_check`. It's inspiration comes from the very excellent [Didn't Test Framework](https://github.com/daviddavis/didnt) and has similar benefits.